global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "2.1.100",
+    "version": "2.1.700",
```

Comment:
**2.1.100**, **2.1.200**, and **2.1.300** have a bug where they can't find the test assembly:

```
========================================
DotnetTest-netcoreapp
========================================
Test run for D:\a\1\s\src\NUnitTestAdapterTests\bin\Release\NUnit.VisualStudio.TestAdapter.Tests.dll(.NETCoreApp,Version=v0.0)
Microsoft (R) Test Execution Command Line Tool Version 15.7.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Cake.exe : The test source file 
"D:\a\1\s\src\NUnitTestAdapterTests\bin\Release\NUnit.VisualStudio.TestAdapter.Tests.dll" provided was not found.
```

**2.1.600** has a buggy (for our purposes) version of VSTest that adds nondeterminism to TRX filenames and breaks our file-by-file uploading of test results when we set the display name of each test in https://github.com/nunit/nunit3-vs-adapter/pull/666.